Resources/Private/GraphQL/schema.root.graphql

```diff
@@ -84,6 +84,8 @@ type Mutation {
     createAssetCollection(title: String!): AssetCollection!
 
     deleteAssetCollection(id: AssetCollectionId!): DeleteAssetCollectionResult!
+
+    updateAssetCollection(id: AssetCollectionId!, title: String): AssetCollection!
```

Comment:
```suggestion
    updateAssetCollection(id: AssetCollectionId!, title: String!): AssetCollection!
```

Reply:
Title would be optional once we have other properties to update, so I didn't want to force it...

Reply:
hm I think it's best to already define everything you want in there and return a `NotImplemented` Exception when it's used without being fully implemented. Then reviews of the API are easier.

Reply:
Dunno, why would we want to transfer extra stuff? E.g. when I only want to set the tags. You had the same logic for editing assets and it kinda made sense...

Reply:
I meant you don't have to make the property required, just fully define what you want to do with the api. Then it makes more sense.

Reply:
What do you mean "fully define what to do with the api"?

Reply:
I had added a lot of definitions to the Schema in the beginning, even though they were not (fully) implemented yet. That made it easier to review the Schema and get an overview.
You said you want to add other properties to the mutation later. So you could just add them now.

Reply:
ah, gotcha. Well, I already implemented it, see the latest code.